src/components/Document/index.js

```diff
@@ -2,19 +2,28 @@ import { h } from 'preact'
 import theme from '../Theme/style.css'
 import style from './style.css'
 
+const DocumentInstructions = ({useCapture}) => {
+  if (useCapture) return <p className={theme.center}>Place it in the rectangle and it will be detected automatically.</p>
```

Comment:
this show or no show logic makes more sense to be done by whoever uses this component. Otherwise you have to pass down more information and you also create more concept dependencies, since this component now needs to know about `useCapture`